plugins/inputs/s7comm/sample.conf

```
@@ -7,6 +7,9 @@
   rack = 0
   slot = 0
 
+  ## connectionType(driveType/driveGroup in s7 protocol) : 1 connect as PG/PC ;2 connect as OP ; 3 connect as S7 basic
+  # connect_type = 2
+
```

Can we please be more user-friendly here by using

```suggestion
  ## Connection or drive type of S7 protocol
  ## Available options are "PD" (programming  device), "OP" (operator panel) or "basic" (S7 basic communication).
  # connection_type = "PD"

```
Please also run `make docs` after changing this!